scalecube-configuration-benchmarks/pom.xml

```diff
@@ -15,7 +15,7 @@
 
   <properties>
     <scalecube-benchmarks.version>1.2.2</scalecube-benchmarks.version>
-    <scalecube-organization.version>2.0.15</scalecube-organization.version>
+    <scalecube-organization.version>2.1.0-SNAPSHOT</scalecube-organization.version>
```

Comment:
this will work?

Reply:
Yes. 

After merging https://github.com/scalecube/scalecube-organization-service/pull/90 to develop, new 2.1.0-SNAPSHOT build of `scalecube-organization-service` is available on `docker hub` and `nexus repository`.
See https://hub.docker.com/r/scalecube/scalecube-organization/tags and https://oss.sonatype.org/#nexus-search;quick~scalecube-organization

Dependency on SNAPSHOT is temporary solution until new version `scalecube-organization-service` is not released (not merged to master).